docpad.coffee

```diff
@@ -66,60 +66,52 @@ module.exports =
 
     # The entire schedule
     schedule: [
-      name: 'Palmer Oliveira',
-      email: 'expalmer@gmail.com',
-      photo: 'themes/yellow-swan/img/palmer.jpg',
-      bio: 'Palmer Oliveira trabalha como Web Developer a mais de 7 anos, é um entusiasta Javascript e estudante de Ciência da Computação na Ulbra Gravataí. Tem uma profunda vontade de aprender e entender como as coisas funcionam.',
-      company: 'Renove3',
+      name: 'Romulo Alves',
+      email: 'me@romuloalv.es',
+      photo: 'themes/yellow-swan/img/romulo.jpeg',
+      bio: 'Engenheiro de Software na PLING, desenvolvendo soluções web de ponta à ponta com JavaScript.',
+      company: 'Pling',
       link:
-        href: 'https://expalmer.github.io',
+        href: 'https://romuloalv.es/',
         text: 'Website'
       presentation:
-        title: 'Conhecendo o Event Loop no Node',
-        description: 'O Event Loop em NodeJs é um dos conceitos mais importantes de se entender para que possamos escrever códigos melhores e tomar proveito de seu famoso I/O não bloqueante. Portanto iremos ver como funciona a implementação do Event Loop na plataforma Nodejs feita pela incrível LIBUV.',
-        date: 'Sábado, 18 de Novembro de 2017',
-        time: '13:00'
+        title: 'Como fazer uma webapp carregar em até 3 segundos',
+        description: 'Como desenvolver uma webapp capaz de carregar em três segundos utilizando Server-side rendering com Next.js e Preact.',
+        date: 'Sábado, 14 de Abril de 2018',
+        time: '14:30'
     ,
-      name: 'Rafael Serpa Nery',
-      email: 'rafael@nery.info',
-      photo: 'themes/yellow-swan/img/rafael-nery.jpg',
-      bio: 'Coordenador de Desenvolvimento de Sistemas na Empresa Libre Soluções de Governo. Envolvido com desenvolvimento de Software desde 2004 com PHP/JS/PostgreSQL. ',
-      company: '',
+      name: 'Paulo Diovani',
+      email: 'paulo@diovani.com',
+      photo: 'https://www.gravatar.com/avatar/eeb1ef7b3ac88503c8189fbecd369387?size=640',
```

Comment:
This photo doesn't have to be 640x640, 120x120 is enough.

https://www.gravatar.com/avatar/eeb1ef7b3ac88503c8189fbecd369387?size=120